### DISCUSSION TITLE
I don't have access to the free student tier when visiting the subscription page

### DISCUSSION THREAD

**Discussion by SpacewaIker** [community/community#19050] [Copilot Conversations] [answered]
On VS Code I started getting notifications that my "Copilot experience is not fully configured" and that I should complete my setup. When clicking on the link, this brings me to the Profile > Settings > GitHub Copilot page where it says that I have to activate copilot.

I saw that as a student I can continue to use Copilot for free as part of the Student Developer Pack. However, when I go to the subscription page for Copilot, I don't get the option to use it for free, as is described [here](https://github.com/pricing#i-am-a-student-can-i-get-access-to-github-copilot-for-free):

> If you are a GitHub Global Campus Student then you will see that GitHub Copilot is offered to you for no charge when you visit the GitHub Copilot subscription page. 

I have double checked and on GitHub Education, it is clear that I have access to the Student Development Pack and that I do not need to reverify my student status.

How can I activate Copilot with the Student Development Pack? Thanks

**Comment by opportunityyourway** [comment]
Same question here

**Reply by Folashy** [reply]
I'm a student of Decagon Institute and could not use the free copilot student developer pack. It is requesting for payment

I'll wait for someone from GitHub staff to help me... I need the access to Copilot. It  has been really helpful to my co students especially for our projects.
![Screenshot 2022-08-22 at 14 31 54](https://user-images.githubusercontent.com/100294216/186088587-8a85e4e1-8118-48cd-8ba2-b691a2f7881d.png)

**Comment by ApidriuC** [comment]
A mi me funciona correctamente teniendo vinculada mi cuenta pro con mi institución educativa.

![image](https://user-images.githubusercontent.com/78039311/174854807-fcabe35f-7725-479e-bc5a-bb52e6244148.png)

Visita el siguiente enlace y visualiza que te sale en empezar prueba gratuita, teniendo tu vinculación deberías poder acceder a este beneficio.

![image](https://user-images.githubusercontent.com/78039311/174854919-9548e816-bc46-4b7b-9ef8-dd337274861f.png)

**Reply by ApidriuC** [reply]
Visita el enlace y me comentas: https://github.com/github-copilot/free_signup

**Reply by opportunityyourway** [reply]
https://python.became.gay/X8B5GJj4eF.png You tell me

**Reply by SpacewaIker** [reply]
Yeah I get the same thing as @opportunityyourway

**Reply by ApidriuC** [reply]
Abajo del botón verde sale continuar sin medio de pago

**Reply by ApidriuC** [reply]
Under the green button you can click to continue without payment method

**Reply by opportunityyourway** [reply]
Yea but that uses the trial, and I don't wanna use that up

**Reply by mozza86** [reply]
i have exactly the same problem when i try to activate copilot it says "GitHub Copilot is now ready to pair!" (https://imgur.com/ZMcIbZ9) but when i check on the billing page it still says "start your free trial" (https://imgur.com/cKvfws7) 
my student plan is still valid (https://imgur.com/GhUhlSo)

**Reply by cheesycod** [reply]
Same issue

**Reply by davidleejy** [reply]
same issue

**Reply by tomas-teston** [reply]
Same issue :(

**Reply by ChrisvanChip** [reply]
Experiencing the same here :c

**Reply by JJTech0130** [reply]
I have the same issue

**Reply by magliocchetti** [reply]
Same issue here....

**Reply by Oct4Pie** [reply]
No luck with the suggestion. Still getting `[ERROR] [default] [2022-06-21T20:13:30.712Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"` and https://github.com/github-copilot/free_signup redirects to the purchase page.

**Reply by ramancini** [reply]
Same issue here, was in the middle of coding and started getting the notification:
> Your Copilot experience is not fully configured, complete your setup.

I hope they fix this soon :/

**Reply by cheesycod** [reply]
Works for me now, try now

**Reply by erenberkaydinc** [reply]
same things for me, I have student pack and can't use github copilot anymore:( I don't know why

**Reply by InvalidSE** [reply]
Same here, anybody found a fix?

**Reply by Jonbroad15** [reply]
This just occured to me today. Is there any fix?

**Comment by Danzo7** [comment]
same here I have student pack and it was working fine yesterday, i which it just a bug and they keep supporting us as students

**Reply by pinheirogh** [reply]
I had the same issue and what I did was to go to the students pack page and hit this button that asks to reverify your academic afilliations. Followed the steps and after a couple minutes the free Copilot was unlocked.
![image](https://user-images.githubusercontent.com/63656610/174892080-a31d492a-208a-4cd2-a9b1-198a051232e6.png)

**Reply by JJTech0130** [reply]
But then I’ll have to submit my proof and wait for it to be processed again?

**Reply by pinheirogh** [reply]
Yeah, I guess ;)

**Reply by magliocchetti** [reply]
When I try to do this it doesn't let me do it because it says that I already verified myself in Nov 2021 and it's not needed.

**Reply by saty24xd** [reply]
happens the same to me

**Reply by minemint** [reply]
I already verified myself but can't use  copilot [
![ภาพ](https://user-images.githubusercontent.com/106004714/174938358-10573495-f673-42e3-ac57-b73225f36176.png)
](url)

**Reply by TudorPal** [reply]
Tried doing this as I have the same problem, but it just says 502: Bad Gateway when I try to access the page...

**Comment by dimdenGD** [comment]
yep doesnt work for me too. also cant reverify my data because i did it recently

**Reply by bloodrain777** [reply]
In phpstorm I'm getting the message 
Your Copilot experience is not fully configured, complete your setup.

**Comment by mengzili** [comment]
Same issue here.

**Comment by saty24xd** [comment]
do you know a solution?

**Comment by masonbesmer** [comment]
+1

**Comment by petersteele111** [comment]
Same issue here. Have student pack for another year still, and I have been in the technical preview as well. Gave me that crap about not being configured today. I went and did the trial shit til august and got it working again, but it does not appear in my github student pack at all. They obviously turned something on today to cause this as a bunch of people are having the same issue. Honestly its bullshit they are charging for it anyways, using our code to autocomplete essentially. So once my student stuff expires I am going to have to pay for Pro and Copilot? Like WTF. Just include it for free with Pro or let beta testers have a year of it or something after it launches publicly in August considering all the help we have given to get it up and running properly.

**Comment by masonbesmer** [comment]
Looks to be fixed everyone, check again here. [https://github.com/github-copilot/free_signup](url)

**Reply by dimdenGD** [reply]
Fixed!

**Reply by Ptolemy2002** [reply]
This link redirects me to the billing section of my settings page without access to copilot.

**Reply by JuCarv-bit** [reply]
It doesn't fix for me either. I have the same problem: this link redirects me to the billing section: [https://github.com/settings/billing](https://github.com/settings/billing)

**Reply by dayglojago** [reply]
This link as posted is currently broken

**Comment by davecheney** [comment, marked as answer]
Sorry for the disruption. Please try again, we’ve made sure that 2020 and 2021 student coupons are now valid for copilot access.

**Reply by Axedyson** [reply]
I've reverifed myself as a student in 2022
<img width="235" alt="image" src="https://user-images.githubusercontent.com/28491857/174965583-182fb96c-2dcf-4d9a-b8f0-f27bcbe1762e.png">

But I still don't have access, could this be because it doesn't currently work for 2022 student coupons?

**Reply by davecheney** [reply]
2020, 2021 and 2022 student tokens are supported. Please contact support.

**Reply by Axedyson** [reply]
Hmm okay it works now, not sure if I just had to wait a bit before it came available, but when I accessed this page: https://github.com/github-copilot/free_signup
it worked!

**Reply by Patrik112358** [reply]
Thanks @AndysonDK, this seems to have fixed it for me too.

@davecheney wouldn't it be helpful if [that page](https://github.com/github-copilot/free_signup) was linked in the https://github.com/settings/copilot section? My thought process is that if it might be necessary to go there for (some) students to get copilot activated, it should perhaps be linked in a convenient place.
Of course, I might have missed something or maybe you are not the person to direct such a question/suggestion to - my apologies if that'd be the case.

**Reply by bloodrain777** [reply]
WTF ? Free version over ?

**Reply by mkafuri** [reply]
Hey, I keep getting this error message "Your Copilot experience is not fully configured, complete your setup." But I already setup everything . I still have free access according to my github billing

**Reply by RaziehSh1987** [reply]
how you solve your problem?

**Reply by matheuskafuri** [reply]
I didn't. The only way to solve it is buying the subscription, I guess...

**Reply by miguel-martinr** [reply]
> Hmm okay it works now, not sure if I just had to wait a bit before it came available, but when I accessed this page: https://github.com/github-copilot/free_signup it worked!

This worked for me :)

I have a student account and I was getting the "Your Copilot experience is not fully configured, complete your setup." message

**Reply by kwabenasapong** [reply]
Hi, i am a student with ALX-SE program, how can I sign on to github copilot?

**Comment by Boldizsarb** [comment]
Same! There is no student discount at all is it?

**Comment by Heath123** [comment]
I had access to the preview, but also have the student pack, and it removed my free access when I should still have it and I can't get it back

![image](https://user-images.githubusercontent.com/13787163/206879062-e7d30884-563c-468e-92b7-23d3701c1f69.png)

**Reply by AsjadSiddiqui** [reply]
I have the same problem :/
<img width="462" alt="image" src="https://user-images.githubusercontent.com/62955947/206879196-9c23994b-e496-4192-8f9c-8f419bc2aa9c.png">

**Reply by asportnoy** [reply]
Same. Just contacted GitHub support about it. Hopefully they'll fix it.

**Reply by ntjess** [reply]
![image](https://user-images.githubusercontent.com/23620506/206879262-5e218fab-8c16-4f69-92ed-daec0ea49147.png)

Same issue here. Commenting for visibility.

**Reply by StianWiu** [reply]
Having the same issue. Hoping this is a bug, as co-pilot is saving me a lot of time working on my assignments.

**Reply by Heath123** [reply]
Seems to be a very recent change, since a lot of people are suddenly coming here

I have tried logging out and in again, and updating the extension but it didn't help

**Reply by taooceros** [reply]
Same

**Reply by busybox11** [reply]
Same issue. Double-checked I still have my pack, I re-entered my information just in case and can confirm, I do have the pack. Hope it'll get fixed soon, it looks like a fairly new change considering everyone seems to have the same problem at the same time.

**Reply by Arisamiga** [reply]
Yea same problem happened to me 🙁

**Reply by terazus** [reply]
Same issue.

**Reply by asportnoy** [reply]
Just got an email from GitHub that my free access has been restored and it's working properly again.

**Reply by AsjadSiddiqui** [reply]
> Just got an email from GitHub that my free access has been restored and it's working properly again.

Which GitHub support email address did you send the email on?

**Reply by smddzcy** [reply]
Same issue.

**Reply by mgar** [reply]
Same issue here :(

**Reply by azaryabernard** [reply]
same issue here.. it's just ended now!

**Reply by Alpereny** [reply]
same here

**Reply by Heath123** [reply]
> Just got an email from GitHub that my free access has been restored and it's working properly again.

Still not working for me

**Reply by AmpedWasTaken** [reply]
have the same issue

**Reply by ntjess** [reply]
It seems like there is a specific GH discussion for technical previews here: https://github.com/orgs/community/discussions/40037#discussioncomment-4234429

Perhaps it would be better to comment there?

**Reply by ozanarmagan** [reply]
I have the same issue

**Reply by Dradows** [reply]
> It seems like there is a specific GH discussion for technical previews here: https://github.com/orgs/community/discussions/40037#discussioncomment-4234429
> 
> Perhaps it would be better to comment there?

already locked

**Reply by atahanoezer** [reply]
I have the same issue

**Reply by pacha2880** [reply]
I have the same issue :(

**Reply by taooceros** [reply]
Seems like it is fixed?

**Reply by Sophon96** [reply]
Same, glad it wasn't just me.

**Reply by Tivikrat** [reply]
https://github.com/github-copilot/free_signup (will be redirected to settings) and retry logging in at your IDE plugin.

**Reply by eliac7** [reply]
Omg, thought I was the only one. Hopefully, this will be fixed soon. I reverified my student pack in July of 2022.

**Reply by AbdulAhadKhan** [reply]
> https://github.com/github-copilot/free_signup (will be redirected to settings) and retry logging in at your IDE plugin.

Doesn't work. Shows me a billing page even though my application is already approved.

**Reply by eliac7** [reply]
Guys, it just got fixed on me, and my profile says that I am eligible for Github Copilot. Check it [here](https://github.com/settings/billing)

![2022-12-11 02_01_55-Billing](https://user-images.githubusercontent.com/26083840/206880160-51cfea25-577a-4ede-a573-52fe1f4b97f6.png)

**Reply by AlexShypula** [reply]
Same issue here! I even went dashboard and found my student pack is still approved and active. It seems like there must be some update that needs to be fixed. I opened up a help ticket, and perhaps more of us should as well?

**Reply by AbdulAhadKhan** [reply]
Fixed!!

**Reply by George-RG** [reply]
> https://github.com/github-copilot/free_signup (will be redirected to settings) and retry logging in at your IDE plugin.

Thanks a lot

**Reply by AlexShypula** [reply]
Looks like it seems fixed for me too 
<img width="617" alt="image" src="https://user-images.githubusercontent.com/36341198/206880190-f8b65128-55ff-443d-8abb-62344bfc35f3.png">

**Reply by Sophon96** [reply]
Fixed!

**Reply by Heath123** [reply]
Can confirm that it works now

**Reply by muhammed-enes-koroglu** [reply]
Did you do something? Or did it just get fixed on its own?

**Reply by thebozturk** [reply]
I still have same issue!

**Reply by eliasblume** [reply]
I got i fixed by re-applying for [github student ](https://education.github.com/pack) even though I already had github student...

 maybe try this if it isent to much work (you will need access to your student mail and a photo of your student ID)

**Reply by Nick-star** [reply]
> Guys, it just got fixed on me, and my profile says that I am eligible for Github Copilot. Check it [here](https://github.com/settings/billing)
> 
> ![2022-12-11 02_01_55-Billing](https://user-images.githubusercontent.com/26083840/206880160-51cfea25-577a-4ede-a573-52fe1f4b97f6.png)

Did you just start the trial period or did you click on settings and it updated?

**Comment by muhammed-enes-koroglu** [comment]
I see people have had this issue last week and for them it has been fixed. I am a student and registered as one at github education. Since today though I can't access the student free edition. Any suggestions?

**Reply by eliasblume** [reply]
I have the same problem, I guess it fixes itself over time ? Still sucks, I hope GitHub fixes this once and for all.

**Comment by XanderStoffels** [comment]
Hi, I've got the same problem. I have a student account and it worked before. Today, it suddenly stopped working while my student status is still valid for a couple more months. Any idea how to fix this?

![image](https://user-images.githubusercontent.com/15852653/211208504-4aa592b8-c98c-4944-bb08-74a888193e8f.png)

**Reply by DevRSC** [reply]
Same Here, Been using it for a while until it suddenly stops working

**Reply by NabilMustofaa** [reply]
same here any solution for these problem? 
i have tried to sign in and sign out on IDE but still cannot use the copilot

**Comment by HikaruChang** [comment]
I had the same problem. It stopped working just now. But my student certification has not expired.

![DB675F5D-ACB5-4F43-8653-BC6D83623935](https://user-images.githubusercontent.com/16264281/211208919-400685e8-bd11-4767-be2a-d1567c9520aa.png)
![E4977FB2-8967-4A7F-BCDC-017B1C9F4C4C](https://user-images.githubusercontent.com/16264281/211208914-d760cc82-7b18-4175-966c-ec3f4d389373.png)

**Reply by FQYQC** [reply]
same here

**Reply by minhhieu184** [reply]
happens the same to me

**Reply by AnandLSB** [reply]
Facing the same issue as well

**Reply by JoshuaMonyei** [reply]
Same here too

**Reply by GreyDGL** [reply]
Same here

**Reply by YoHoSo** [reply]
same here, it was working just fine yesterday and stopped working this morning

**Reply by HikaruChang** [reply]
I received an email, which has only been renewed for one month

![1B6A4AEF-55B5-4945-8B76-C3696F2A3EF0](https://user-images.githubusercontent.com/16264281/211234121-22c27fd8-81e8-487e-a44f-ccd2c14e3491.jpeg)
![3751589B-B204-44C4-8EC3-C43AA71D67E3](https://user-images.githubusercontent.com/16264281/211234124-002708e1-89ee-4a83-8eee-ee6c51617041.jpeg)

**Reply by HikaruChang** [reply]
I seem to have found the answer.
![17D26F6C-8531-4D4E-B689-3B48B323FF04](https://user-images.githubusercontent.com/16264281/211235513-57d393d7-064c-4986-b48c-c9bed1e2629f.jpeg)

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by zxrohex** [comment]
I have the same issue currently.

**Comment by Susansansan** [comment]
I have the same issue currently.

**Comment by xup6m6fu04** [comment]
I have the same issue currently.

**Comment by JuanAntonioV** [comment]
![image](https://user-images.githubusercontent.com/96824210/211239103-f2587df4-8892-4ab7-8261-4e30965ba2c7.png)

the comeback... i don't understand why? but it says, GitHub Copilot is free for maintainers of popular open source software on GitHub and verified students. and I'm still enrolled in the Student Developer Pack.

**Comment by otkrickey** [comment]
Solved: #43634

**Comment by razvyrazvy41** [comment]
i have the same issue ! please help!

**Comment by BilalTariq01** [comment]
Same Issue. But after disabling and enabling the vscode Copilot extension it starts working.

**Comment by Nick-star** [comment]
I have the same issue

**Comment by Ademking** [comment]
Same issue

**Reply by huuthanhnguy** [reply]
Disable and enable copilot viscose extension and it will start working.

**Comment by eeantoss** [comment]
Same issue

**Reply by huuthanhnguy** [reply]
Disable and enable copilot viscose extension and it will start working.

**Comment by yatinj30** [comment]
Same issue

**Reply by huuthanhnguy** [reply]
Disable and enable copilot vscode extension and it will start working.

**Comment by huuthanhnguy** [comment]
Disable and enable copilot viscose extension and it will start working.

**Reply by huuthanhnguy** [reply]
Disable and enable copilot viscose extension and it will start working.

**Comment by mirceamaierean** [comment]
Same issue

**Comment by bt** [comment]
I just suddenly lost access a couple hours ago. Hopefully it will fix itself in a couple hours...

**Reply by h1dd3nsn1p3r** [reply]
Same issue here.

**Comment by sena-nana** [comment]
My permission was restored on the 9th, but today it suddenly expired again...

**Comment by Mitrano-sensei** [comment]
Same issue here :c

**Comment by Neel-shetty** [comment]
I registered recently and copilot is paid for me too, can you allow 2023 students?

**Comment by LucasNavarroB** [comment]
Hello, I wanted to start using Copilot and when I log in, it sends me the payment page, and as I understand it, it is free for verified students.
Im verifired student since 2021.
![Captura de pantalla 2023-04-27 151054](https://user-images.githubusercontent.com/49734933/234968188-721d64ab-3ec5-4d2e-8279-571cd1ac8bc9.png)

**Reply by davecheney** [reply]
@LucasNavarroB you must apply the student discount coupon you received to your github account, then your copilot student access will be unlocked.
